examples/profile-client/src/main/java/com/palantir/example/profile/schema/ProfileSchema.java

```diff
@@ -44,46 +44,69 @@ private static Schema generateSchema() {
                 Namespace.DEFAULT_NAMESPACE,
                 OptionalType.JAVA8);
 
-        schema.addTableDefinition("user_profile", new TableDefinition() {{
-            rowName();
+        schema.addTableDefinition("user_profile", new TableDefinition() {
```

Comment:
I would rather revert the changes in this file, and suppress the checkstyle warnings for RightCurly and Indentation:
```
@SuppressWarnings({"checkstyle:Indentation","checkstyle:RightCurly"})
```